README.md

```diff
@@ -1,6 +1,7 @@
 # React Inline Confirm Component
 
-> A react component that allows you to get a confirmation of action from the user before executing an action.
+> A react component that allows you to get a confirmation of action from the user before executing an action. 
+> 'isExecuting' is the state of react when the button was clicked. The default value should be false. It can be passed as an argument for your react component.
```

Comment:
Don't put this in the tagline. I would move this under the _How to Use_ section.

Also use backticks `` around isExecuting to make it format as code.